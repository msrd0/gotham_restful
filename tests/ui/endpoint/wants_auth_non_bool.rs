#[macro_use]
extern crate gotham_restful;

#[derive(Resource)]
#[resource(read_all)]
struct FooResource;

#[read_all(wants_auth = "yes, please")]
async fn read_all() {}

fn main() {}
