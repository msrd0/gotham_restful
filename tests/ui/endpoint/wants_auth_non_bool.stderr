error: Expected boolean literal
 --> $DIR/wants_auth_non_bool.rs:8:25
  |
8 | #[read_all(wants_auth = "yes, please")]
  |                         ^^^^^^^^^^^^^

error[E0412]: cannot find type `read_all___gotham_restful_endpoint` in this scope
 --> $DIR/wants_auth_non_bool.rs:5:12
  |
5 | #[resource(read_all)]
  |            ^^^^^^^^ not found in this scope
